docstr_coverage/coverage.py

```diff
@@ -211,6 +215,8 @@ def print_docstring(base, node, filename, ignore_names=()):
                 docs_needed -= 1
             elif skip_class_def and "_" not in name and (name[0] == name[0].upper()):
                 docs_needed -= 1
+            elif skip_private and name.startswith("_"):
```

Comment:
Can you please add a check to prevent catching mangled names here? I think adding `and not name[1].startswith("_")` should probably do the trick

Reply:
Ahh sorry you also thought of that already :)

Reply:
Will do!

Reply:
Fixed in 3bc8897

Reply:
I suspect Black is complaining about the use of single quotes in your recent commit

Reply:
oops! I normally use single quotes but was doing my best to follow Black style for this repo... my hand must have slipped :)

Reply:
Quotes fixed in 4877f4d